### ISSUE TITLE
[BUG] AttributeError: 'Index' object has no attribute '_ndarray_values'

### ISSUE THREAD

**Issue by Jinwen321** [sktime/sktime#1556]
Dear all,

I am new to sktime. When I followed the forecasting part of the tutorial and ran this code line: from sktime.utils.plotting import plot_series, I got this error: AttributeError: 'Index' object has no attribute '_ndarray_values'. The detailed traceback were:

  File "<ipython-input-15-c9c95ff1127e>", line 1, in <module>
    from sktime.utils.plotting import plot_series

  File "D:\Anaconda3\lib\site-packages\sktime\utils\plotting.py", line 20, in <module>
    from sktime.datatypes import convert_to

  File "D:\Anaconda3\lib\site-packages\sktime\datatypes\__init__.py", line 6, in <module>
    from sktime.datatypes._check import check_is, check_raise, mtype

  File "D:\Anaconda3\lib\site-packages\sktime\datatypes\_check.py", line 31, in <module>
    from sktime.datatypes._panel import check_dict_Panel

  File "D:\Anaconda3\lib\site-packages\sktime\datatypes\_panel\__init__.py", line 7, in <module>
    from sktime.datatypes._panel._examples import example_dict as example_dict_Panel

  File "D:\Anaconda3\lib\site-packages\sktime\datatypes\_panel\_examples.py", line 60, in <module>
    X = X.set_index(["instances", "timepoints"])

  File "D:\Anaconda3\lib\site-packages\pandas\core\frame.py", line 4295, in set_index

  File "D:\Anaconda3\lib\site-packages\pandas\core\indexes\base.py", line 3902, in __contains__
    left_lev_indexer = ensure_int64(left_lev_indexer)

  File "pandas\_libs\properties.pyx", line 34, in pandas._libs.properties.CachedProperty.__get__

  File "D:\Anaconda3\lib\site-packages\pandas\core\indexes\base.py", line 608, in _engine
    The array interface, return my values.

AttributeError: 'Index' object has no attribute '_ndarray_values'

I have no idea how that error happened. Could anyone kindly help me? Thanks a lot!

**Comment by jbbqqf** [comment]
Hi! Triaging older issues this week — I couldn't reproduce this on the current `main`.

`'Index' object has no attribute '_ndarray_values'` came from `pandas._ndarray_values` (a private attribute removed in pandas 0.25.x). sktime no longer references it anywhere:

- [`sktime/utils/plotting.py`](https://github.com/sktime/sktime/blob/main/sktime/utils/plotting.py) — the file from your traceback; no `_ndarray_values` access remains.
- `grep -rn "_ndarray_values" sktime/` on commit `bd8fe08` returns zero matches.

A pinned modern stack (`pandas>=2`, current sktime) does not reproduce this error.

If you're still hitting it on a recent version, would you mind sharing a fresh repro with `sktime --version` / `pandas --version`? Otherwise I'd suggest closing — happy to be wrong if I missed something.

(Disclosure: I drafted this comment with help from Claude Code while triaging stale issues; the references above were verified manually against the current `main` branch at commit `bd8fe08`.)

**Comment by benHeid** [comment]
I agree that this issue seems not to be reproducible anymore. I can also confirm that no references to _ndarray_values can be found

<details>

```python
import importlib
import pathlib
import sktime

print(f"sktime version: {sktime.__version__}")

# Original repro: importing plot_series triggered a chain of imports
# that eventually hit pandas Index._ndarray_values (removed in pandas 0.25+).
try:
    module = importlib.import_module("sktime.utils.plotting")
except AttributeError as e:
    raise AssertionError(
        f"Importing sktime.utils.plotting raised AttributeError (bug present): {e}"
    )

assert hasattr(module, "plot_series"), (
    "sktime.utils.plotting imported but plot_series is missing"
)
print("Import of sktime.utils.plotting succeeded and plot_series is available.")

# Cross-check the maintainer's second claim: no remaining references to the
# removed private pandas attribute anywhere in the installed sktime source.
sktime_root = pathlib.Path(sktime.__file__).parent
offending_files = []
for py_file in sktime_root.rglob("*.py"):
    try:
        text = py_file.read_text(encoding="utf-8", errors="ignore")
    except OSError:
        continue
    if "_ndarray_values" in text:
        offending_files.append(py_file)

print(f"Files referencing '_ndarray_values': {len(offending_files)}")
assert not offending_files, (
    f"Found references to removed pandas attribute '_ndarray_values' in: "
    f"{offending_files}"
)

print("No references to the removed pandas private attribute were found.")
```

</details>


Triaged with Claude Code
